vllm/worker/hpu_model_runner.py

```diff
@@ -584,6 +605,8 @@ def __init__(
         self.bucketing_global_state = HPUBucketingGlobalState()
         self._setup_buckets()
         self._set_gc_threshold()
+        self.use_contiguous_pa = os.environ.get('VLLM_CONTIGUOUS_PA',
```

Comment:
self.use_contiguous_pa = os.environ.get('VLLM_CONTIGUOUS_PA', 'true').lower() in [1, ''true'] 
and we can define a function like get_env() to replace others